package.json

```diff
@@ -2,8 +2,23 @@
   "name": "app",
   "version": "1.0.0",
   "description": "UBCEA lounge app backend",
-  "main": "index.js",
+  "main": "app.js",
+  "type": "module",
   "devDependencies": {
-    "prettier": "3.3.3"
+    "chai": "^5.1.1",
+    "mocha": "^10.7.3",
+    "prettier": "3.3.3",
+    "supertest": "^7.0.0"
+  },
+  "dependencies": {
+    "body-parser": "^1.20.3",
+    "dotenv": "^16.4.5",
+    "express": "^4.20.0",
+    "moment-timezone": "^0.5.45",
+    "pg": "^8.13.0"
```

Comment:
We should leave this package.json for global node configurations. The tests folder for the API's should live in the `app` directory ie. `app/tests`. This way we don't have to add the node dependencies twice.

Reply:
Moved tests into app directory, running `npm test` from the root directory still requires all the dependencies for the CI, check out https://github.com/ubcesports/lounge-hub/actions/runs/10945602636/job/30390194369 failure.